jfrog_platform_cloud/manifest.json

```diff
@@ -0,0 +1,65 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "798102cb-6c52-4a16-bc1b-48c2e6b54e71",
+  "app_id": "jfrog-platform-cloud",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View and analyze JFrog Artifactory Cloud logs",
+    "title": "JFrog Platform Cloud",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "JFrog Artifactory Logs dashboard - HTTP Requests",
+        "image_url": "images/jfrog_platform_cloud_logs_0.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Artifactory Logs dashboard - Request Logs",
+        "image_url": "images/jfrog_platform_cloud_logs_1.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Artifactory Logs dashboard - Operations",
+        "image_url": "images/jfrog_platform_cloud_logs_2.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Log Collection",
+      "Offering::Integration",
+      "Submitted Data Type::Logs",
+      "Supported OS::Linux",
+      "Category::Kubernetes",
+      "Category::Containers",
+      "Category::Security"
+    ]
+  },
+  "assets": {
+    "dashboards": {
+      "JFrog Platform Cloud Log Analytics\n": "assets/dashboards/jfrog_platform_cloud_overview.json"
+    },
+    "monitors": {},
+    "logs": {},
+    "saved_views": {},
+    "integration": {
+      "source_type_name": "JFrog Platform Cloud",
+      "configuration": {},
+      "events": {
+        "creates_events": false
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    }
+  },
```

Comment:
You'll need to add a "metrics" section like this - https://github.com/DataDog/integrations-extras/blob/master/flagsmith/manifest.json#L38-L42 or remove the `metadata.csv` file if your integration doesn't emit metrics.

Reply:
`metadata.csv` is removed.